functions.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+function intToBritDate(string $intDate): string {
+    $timestamp = strtotime($intDate);
+    return date("d.m.Y", $timestamp);
+}
+
+function albumOrSingleConvert(string $albOrSin): string {
+    if ($albOrSin == 'a') {
+        return 'Album';
+    } else {
+        return 'Single';
+    }
+}
+
+function reasonToSymbol(string $reason): string {
+    if ($reason == 'audio') {
+        return '<i class="fa-solid fa-music"></i>';
+    } elseif ($reason == 'visual') {
+        return '<i class="fa-solid fa-eye"></i>';
+    } else {
+        return '<i class="fa-solid fa-music"></i> <i class="fa-solid fa-eye"></i>';
+    }
+}
+
+function noImageIconIfNoImageInDB(?string $link): string {
+    if (is_null($link)) {
+        return 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSrJgwdOAjqaZGS7kn35IVm_ZN6E4XFuJ7V_g&usqp=CAU';
+    } else {
+        return $link;
+    }
+}
+
+function altTagMaker(Album $anAlbum): string {
+    if (is_null($anAlbum->getLinkToImage())) {
+        return 'no image in database';
+    } else {
+        return $anAlbum->getName() . 'album cover';
+    }
+}
+
+function albumObjectToHtml(Album $anAlbum): string {
```

Comment:
This function will need some unit tests as well, making sure it returns the correct HTML for a given album